buildchain/buildchain/config.py

```diff
@@ -9,6 +9,7 @@
 
 import os
 import shlex
+import enum
```

Comment:
Commit message could be more explicit, like:

```
build: add a test to check for external binaries

This commit introduces a new set of tasks to check for the presence of some required binaries.
If a required binary is missing, the task will fail and stop the build with a clear error message (instead of a big stacktrace).
```

Otherwise, the code LGTM.